sqldelight-gradle-plugin/src/test/integration-multiplatform/src/commonMain/sqldelight/com/squareup/sqldelight/integration/NullableTypes.sq

```diff
@@ -14,4 +14,4 @@ FROM nullableTypes;
 exprOnNullableColumn:
 SELECT *
 FROM nullableTypes
-WHERE val2 = ?;
\ No newline at end of file
+WHERE val2 IS ?;
```

Comment:
@AlecStrong I think this is also a good example of the issue. Looking at the SQL, I would think that if I passed in `null` it would never evaluate to `1`, but it would. And the converse if I was looking at the test that uses this is that I would expect the query to be using `IS`